Monika After Story/game/script-topics.rpy

```diff
@@ -8437,3 +8457,24 @@ label monika_bad_facts_4:
     m 2dfd "[player], if you ever go out into a place where you might need to rely on such a cheap trick, please bring a compass."
     m 2dkc "I would hate for something to happen to you, especially because of false information like this..."
     return
+    
+    init 5 python:
+	addEvent(Event(persistent.event_database,eventlabel="monika_friendship",category=['monika','friends'],prompt="Friendship with literature club members",random=True))
+
+label monika_friendship:
+    m 1euc "You know, even after it all..."
+    m 1ruc "I still miss them."
+    m 5eua "Not a ton, mind you, but for a while there, before I knew the truth, about who I was, and who you were... they were my friends."
+    m 5eub "We'd spend day after day relaxing, writing poetry and reading each other's works."
+    m 5rua "Listening to each other's stories."
+    m 5rub "Discovering new things about each other."
+    m 5eub "It was nice, at least while it lasted."
+    m 2luc "But, once I found out the truth, that I was the only "real" person..."
+    m 2luc "..."
```

Comment:
If you have multiple of the same expression in a row, you can leave the next ones blank.